### ISSUE TITLE
The output is ignoring path in RequestMapping

### ISSUE THREAD

**Issue by JasonQuinn** [kongchen/swagger-maven-plugin#315]
When I generate the json file the output should be /userstore/clients/{clientId}/customers/{externalCustomerId} but it is leaving out the externalCustomerId part.

```
@Api(value = "Customer")
@RestController()
@RequestMapping("/userstore/clients/{clientId}/customers")
public class CustomersController {

  @ApiOperation(value = "Retrieve customer")
  @RequestMapping(
          path = "/{externalCustomerId}",
          method = RequestMethod.GET
  )
  public Customer retrieveCustomer(
          @PathVariable(value = "clientId") long clientId,
          @PathVariable(value = "externalCustomerId") String externalCustomerId
  ) throws StoreAccessException {
    return customerStore.retrieveCustomer(clientId, externalCustomerId);
  }
```

Output 

```
"/userstore/clients/{clientId}/customers" : {
      "get" : {
        "tags" : [ "Customer" ],
        "summary" : "Retrieve customer",
        "description" : "",
        "operationId" : "retrieveCustomer",
        "parameters" : [ {
          "name" : "clientId",
          "in" : "path",
          "required" : true,
          "type" : "integer",
          "format" : "int64"
        }, {
          "name" : "externalCustomerId",
          "in" : "path",
          "required" : true,
          "type" : "string"
        } ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Customer"
            }
          }
        }
      }
    },
```

**Comment by Interregnum** [comment]
@JasonQuinn :
I had the same problem. It was fixed by replacing (path = "/something") with (value = "/something").

**Comment by cforce** [comment]
Its no fix .. its a workaround.  Please fix this bug, as both params mus be supported. Btw. i confirm this BUG

**Comment by tjuchniewicz** [comment]
Looks like fixed in 3.1.5-SNAPSHOT. @JasonQuinn could you verify?
